bitnami/fluentd/values.yaml

```diff
@@ -0,0 +1,258 @@
+## Global Docker image parameters
+## Please, note that this will override the image parameters, including dependencies, configured to use the global value
+## Current available global Docker image parameters: imageRegistry, imagePullSecrets and storageClass
+##
+# global:
+#   imageRegistry: myRegistryName
+#   imagePullSecrets:
+#     - myRegistryKeySecretName
+#   storageClass: myStorageClass
+
+## Bitnami Fluentd image version
+## ref: https://hub.docker.com/r/bitnami/fluentd/tags/
+##
+image:
+  registry: docker.io
+  repository: bitnami/fluentd
+  tag: 1.7.2-debian-9-r8
+  ## Specify a imagePullPolicy
+  ## Defaults to 'Always' if image tag is 'latest', else set to 'IfNotPresent'
+  ## ref: http://kubernetes.io/docs/user-guide/images/#pre-pulling-images
+  ##
+  pullPolicy: IfNotPresent
+  ## Optionally specify an array of imagePullSecrets.
+  ## Secrets must be manually created in the namespace.
+  ## ref: https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/
+  ##
+  # pullSecrets:
+  #   - myRegistryKeySecretName
+
+## String to partially override fluentd.fullname template (will maintain the release name)
+##
+# nameOverride:
+
+## String to fully override fluentd.fullname template
+##
+# fullnameOverride:
+
+## Cluster Domain
+clusterDomain: cluster.local
+
+forwarder:
+  ## Name of the config file that will be used by Fluentd at launch
+  ## Fluentd will look for it under the /opt/bitnami/fluentd/conf directory
+  ##
+  configFile: fluentd.conf
+
+  ## Name of the configMap that contains the configuration files for fluentd
+  ## If not specified, one will be created by default
+  # configMap:
```

Comment:
I would make the configMap value to be evaluated as a template. Very useful for umbrella charts